portfolio/src/main/webapp/index.html

```diff
@@ -1,34 +1,61 @@
 <!DOCTYPE html>
```

Comment:
nit: I would've preferred the prettier changes to be in a separate PR on their own, but its few enough files that this is fine.